About continuous deployment (#5771)
dbt-labs/docs.getdbt.com: website/docs/docs/deploy/continuous-integration.md

<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/example-smart-cancel-job.png" width="70%" title="Example of an automatically canceled run"/>

### Run slot treatment <Lifecycle status="team,enterprise" />

> CI runs do not consume run slots. This guarantees a CI check will never block a production run.

Reviewer: ```suggestion
CI runs don't consume run slots. This guarantees a CI check will never block a production run.
```
